sky/utils/common_utils.py

```diff
@@ -697,3 +697,26 @@ def truncate_long_string(s: str, max_length: int = 35) -> str:
     if len(prefix) < max_length:
         prefix += s[len(prefix):max_length]
     return prefix + '...'
+
+
+def get_gpu_resource_name(name: str = 'nvidia.com/gpu'):
```

Comment:
Instead of having name as a arg, we probably want to move the hardcoded string to the top of kubernetes_utils.py as `DEFAULT_GPU_RESOURCE = 'nvidia.com/gpu'` and use that in the method if the envvar does not exist.
```suggestion
def get_gpu_resource_name():
```

Reply:
done